api/openapi.gen.yaml

```diff
@@ -83,6 +83,29 @@ paths:
           $ref: "#/components/responses/NotFound"
         '500':
           $ref: "#/components/responses/InternalError"
+  /instance_types/{source_id}:
+      get:
+        description: 'Return a list of instance types'
+        operationId: getListInstanceTypes
+        parameters:
+          - name: source_id
+            in: path
+            required: true
+            description: 'Database ID of a source.'
```

Comment:
This is the source database ID, is it? Can you make the description more clear that this is actually ID from the sources database, not ours?